llvm/test/CodeGen/AIE/aie2/schedule/resource/vst_pack.mir

```diff
@@ -0,0 +1,33 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+#
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+# RUN: llc -march=aie2 -run-pass=postmisched %topdown-multi %s -o - | FileCheck %s
```

Comment:
Nit: should use -mtriple, see https://github.com/llvm/llvm-project/commit/806761a7629df268c8aed49657aeccffa6bca449
(we should apply a mass-fix for the existing tests at some point)